lia-core/src/main/java/com/lithium/community/android/manager/LiClientManager.java

```diff
@@ -149,6 +149,27 @@ public static LiClient getUserSubscriptionsClient(LiClientRequestParams params)
                 LiQueryConstant.LI_SUBSCRIPTTION_QUERYSETTINGS_TYPE, LiSubscriptions.class);
     }
 
+    /**
+     * Retains a Rest GET client which retrieves all subscriptions by a user to a message
+     * @param params - the client params an instance of
+     * {@link com.lithium.community.android.model.request.LiClientRequestParams.LiUserMessageSusbscriptionRequestParans}
+     * @return - a rest client
+     * @throws LiRestResponseException
+     */
+    public static LiClient getUserMessageSubscriptionsClient(LiClientRequestParams.LiUserMessageSusbscriptionRequestParans params)
+            throws LiRestResponseException {
+        params.validate(Client.LI_USER_MESSAGE_SUBSCRIPTIONS_CLIENT);
+        LiQueryValueReplacer replacer = new LiQueryValueReplacer();
+        replacer.replaceAll("##", params.getMessageId());
+        replacer.replaceAll("&&", params.getUserId());
```

Comment:
Can create constants for these "##","&&"?

Reply:
For all other queries this is the standard used.